build/Nuget/Microsoft.NET.Sdk.Nuget.targets

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <Target Name="LayoutSdkPackage">
+  <Target Name="LayoutSdkPackage" BeforeTargets="Build" >
```

Comment:
I'll change to Build to DependsOnTarget=LayoutSdkPackage. The build still requires the package layout to be constructed for the nuspec to pick it up, unless there is some change there that I'm missing?